orttraining/orttraining/python/experimental/checkpoint.py

```diff
@@ -29,9 +29,10 @@ def experimental_state_dict(ort_trainer):
             torch_state[n.name] = torch.from_numpy(np.array(onnx.numpy_helper.to_array(n)))
 
     # Need to remove redundant initializers and name suffices to map back to original torch state names
```

Comment:
we are not removing name suffices here are we?

Reply:
fixed